doc/sphinx/source/python_api/_autosummary/ngraph.ops.rst

```diff
@@ -76,6 +77,7 @@ ngraph.ops
       replace_slice
       reshape
       reverse
+      RNNCell
```

Comment:
Why are we not following the case convention here? We should have `rnn_cell` in this list, not `RNNCell`